src/Certificate.php

```diff
@@ -688,6 +688,17 @@ public static function cronCertificate($task = null)
         $message      = [];
         foreach (array_keys(Entity::getEntitiesToNotify('use_certificates_alert')) as $entity) {
             $before = Entity::getUsedConfig('send_certificates_alert_before_delay', $entity);
+            $repeat = Entity::getUsedConfig('certificates_alert_repeat', $entity);
+            if ($repeat > 0) {
+                $where_date = [
+                    'OR' => [
+                        ['glpi_alerts.date' => null],
+                        ['glpi_alerts.date' => ['<', new QueryExpression('CURRENT_TIMESTAMP() - INTERVAL ' . $repeat . ' second')]],
```

Comment:
You will have to delete the alert before being able to create a new one, in order to respect unicity rule ``` UNIQUE KEY `unicity` (`itemtype`,`items_id`,`type`) ``` .